docker-compose.yml

```diff
@@ -82,6 +82,18 @@ services:
     volumes:
       - ./microservices/http_validator:/home/node/app
     command: "npm start"
+  ping-verification:
```

Comment:
@MatiasMinchaca why you are using a Dockerfile if the web verifications service doesn't need?

```
web-verification:
    image: node:18 
    user: "node"
    working_dir: /home/node/app
    expose:
      - "80"
    volumes:
      - ./microservices/http_validator:/home/node/app
    command: "npm start"
```

Reply:
use a dockerfile because the default node image does not contain the 'ping' command necessary for verification